package.json

```diff
@@ -15,21 +15,28 @@
     "postinstall": "husky install",
     "test-e2e": "playwright test"
   },
+  "dependencies": {
+    "@hookform/resolvers": "^3.3.1",
+    "classnames": "^2.3.2",
+    "react-hook-form": "^7.46.2",
+    "react-to-print": "^2.14.13",
+    "zod": "^3.22.2"
+  },
   "devDependencies": {
     "@openmrs/esm-framework": "next",
     "@playwright/test": "1.44.0",
     "@swc/cli": "^0.1.62",
     "@swc/core": "^1.3.89",
     "@swc/jest": "^0.2.29",
-    "@testing-library/dom": "^9.3.3",
-    "@testing-library/jest-dom": "^6.1.5",
-    "@testing-library/react": "^14.1.2",
-    "@testing-library/user-event": "^14.5.1",
+    "@testing-library/dom": "^10.1.0",
+    "@testing-library/jest-dom": "^6.4.5",
+    "@testing-library/react": "^15.0.7",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
Wound up needing to bump these to fix [this warning](https://github.com/search?q=+Warning%3A+%60ReactDOMTestUtils.act%60+is+deprecated+in+favor+of+%60React.act%60.+Import+%60act%60+from+%60react%60+instead+of+%60react-dom%2Ftest-utils%60.+See+https%3A%2F%2Freact.dev%2Fwarnings%2Freact-dom-test-utils+for+more+info.&type=issues).